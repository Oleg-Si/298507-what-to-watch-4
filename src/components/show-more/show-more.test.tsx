import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ShowMore from './show-more';

describe(`Проверяет снепшоты компонента ShowMore`, () => {
  it(`Проверяет снепшот компонента ShowMore с кнопкой`, () => {
    const tree = renderer.create(
        <ShowMore
          allFilmsCount={8}
          filmsCount={5}
          onShowMoreClick={() => null}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Проверяет снепшот компонента ShowMore без кнопки`, () => {
    const tree = renderer.create(
        <ShowMore
          allFilmsCount={8}
          filmsCount={10}
          onShowMoreClick={() => null}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
