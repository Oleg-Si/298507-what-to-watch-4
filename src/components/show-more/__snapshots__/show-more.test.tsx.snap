// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверяет снепшоты компонента ShowMore Проверяет снепшот компонента ShowMore без кнопки 1`] = `
<div
  className="catalog__more"
/>
`;

exports[`Проверяет снепшоты компонента ShowMore Проверяет снепшот компонента ShowMore с кнопкой 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
